.github/dependabot.yml

```diff
@@ -49,3 +51,130 @@ updates:
       docker:
         patterns:
         - "docker/*"
+  ## Currently dependabot does not support wildcard or multiple directories
```

Comment:
@hairyhum Does it support multiple stanzas / entries for the same package ecosystem? that is multiple `package-ecosystem: "docker"` ?

Reply:
It's not officially documented, but people use it like that.

Reply:
https://github.com/dependabot/dependabot-core/issues/2178